pyro/infer/mcmc/hmc.py

```diff
@@ -28,31 +29,49 @@ class HMC(TraceKernel):
     :param int num_steps: The number of discrete steps over which to simulate
         Hamiltonian dynamics. The state at the end of the trajectory is
         returned as the proposal.
+    :param dict transforms: Optional dictionary that specifies a transform
+        for a sample site with constrained support to unconstrained space. The
+        transform should be invertible, and implement `log_abs_det_jacobian`.
+        If not specified and the model has sites with constrained support,
+        automatic transformations will be applied, as specified in
+        ``torch.distributions.constraint_registry``.
```

Comment:
Pyro docs now link directly to PyTorch documentation. Can you update this link to
```
:module:`torch.distributions.constraint_registry`
```
(and maybe `make docs` and ensure the link works)

Reply:
Great! Will do.